engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OAuth2Configuration.scala

```diff
@@ -19,10 +19,10 @@ case class OAuth2Configuration(usersFile: URI,
                                clientSecret: String,
                                clientId: String,
                                profileUri: URI,
-                               profileFormat: Option[ProfileFormat],
+                               profileFormat: Option[ProfileFormat] = Some(OIDC),
```

Comment:
Can we have default only in one place: `defaultUiConfig.conf`? Discovering the code to check what is default value for given property is painful. Also shouldn't be added sth to our configuration docs?

Reply:
Reverted.